pkgs/development/node-packages/node-packages.json

```diff
@@ -7,6 +7,7 @@
 , "@bitwarden/cli"
 , "@commitlint/cli"
 , "@commitlint/config-conventional"
+, "@emacs-eask/eask"
```

Comment:
Ok, this was slightly confusing for me (first time I see this format). It seems that this generates a `nodePackages."@emacs-eask/eask"` instead of `nodePackages.eask` as I expected initially. This is clear a non-standard name, but it can still be referenced.

However I think it would be better to give a proper package name. Taking the example `@bitwarden/cli`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/overrides.nix#L91-L93

I think you should also do something similar for your package.

Reply:
BTW, in this case you probably don't need the override. Similar to this: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/overrides.nix#L56

Reply:
Actually the `name` override is a good idea:
```
nix-repl> legacyPackages.aarch64-linux.nodePackages."@emacs-eask/eask".name
"_at_emacs-eask_slash_eask-0.6.43"
```

Reply:
Just to confirm, so you want me to rename the package to `@emacs-eask/cli`?

Reply:
No, what I am saying is that you should give a proper name for the package, e.g. `eask` instead of `@emacs-eask/eask`.